xwf/gateway/deploy/roles/pkgrepo/files/jfrog.pub

```diff
@@ -0,0 +1,30 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
```

Comment:
There is a link to directly get the public [gpg key](https://facebookconnectivity.jfrog.io/artifactory/api/gpg/key/public ) can you use it in order to not host it in the codebase and dynamically pull it ? 

`wget -qO - https://facebookconnectivity.jfrog.io/artifactory/api/gpg/key/public | apt-key add -` something like that should do the trick (if wget and gnupg* is installed)

Reply:
its done